RELEASING.md

```diff
@@ -1,89 +1,96 @@
 # Release Process
+
 We have a goal of simultaneously serving multiple versions of the docs. This
-means that there are multiple versions of the docs existing next to each other
-in the docs repo. This differs from what is normal, where only a single version
+means that multiple versions of the docs exist in sibling directories in the docs repo. This differs from what is normal, where only a single version
 exists in the repo at a time and a git tag is created to reference old versions.
 
 The versions that exist together are major or minor versions (e.g. v1.1, v1.2
 and v2.0). There only exists a single patch version (or pre-release
 alphas, betas and release candidates) for each major/minor version. When
 we do a new patch release, the updates are made in-place.
 
-## Preparing for a release
+## Types of Releases
+
+There are four types of releases, each with a slightly different release process:
+
+1. First release candidate for the next minor release (e.g. `vX.Y.0-rc1`)
+2. Subsequent release candidates for the next minor release (e.g. `vX.Y.0-rcA` where `A > 1`)
+3. Promoting a release candidate to a final release (e.g. `vX.Y.0`)
+4. Patch release (e.g. `vX.Y.Z` where `Z > 0)
+
+## 1. First release candidate for the next minor release
+
+To cut the first release candidate, we will need to do three things:
+
+1. Select and set versions of subcomponents.
+2. Build and push images for `calico/node`.
+3. Update the calico docs.
+
+Since this is by far the longest release process, the steps are split up by section so that they are easier to follow. But (for now) they must be followed in this precise order!
+
+### calico/node
+
 1. Ensure master docs are passing semaphore tests.
 1. Check out latest master locally.
+1. Run `./release-scripts/do_release.py` to copy master directory to the new version directory. Commit
+these new files in their own commit.
+   > Note: The remaining changes below can all go in the same, separate commit.
+1. Edit `_data/versions.yml`:
+   1. Add a new section **at the end of the file**. (If a release is added to the top of the file it will become the 'Latest Release'). Be sure to add the date of the release in the description.
+   1. Bump `currentReleaseStream` to this new release stream (e.g. `vX.Y`). 
+1. Run `make -C calico_node release` to create the new `calico/node` images and release tar. 
+Once complete, follow the instructions to push the new images.
+
+### Docs
+
+1. Add a section in `_config.yml` so that `page.version` will be set correctly in the new subdirectory.
```

Comment:
This is less expressive than it used to be (used to have an example) - feels like this is a step backwards in terms of understandability for someone new.